rasa/cli/train.py

```diff
@@ -144,3 +159,58 @@ def extract_additional_arguments(args: argparse.Namespace) -> Dict:
         arguments["debug_plots"] = args.debug_plots
 
     return arguments
+
+
+def _enrich_config(
+    config_path: Text, missing_keys: List[Text], FALLBACK_CONFIG_PATH: Text
```

Comment:
`FALLBACK_CONFIG_PATH ` is shadowing the imported `FALLBACK_CONFIG_PATH `

Reply:
good point 👍